sys-power/throttled/files/throttled

```diff
@@ -0,0 +1,5 @@
+#!/sbin/openrc-run
+command="/usr/bin/throttled"
+pidfile="/var/run/throttled.pid"
```

Comment:
using /run would be just fine, as /var/run is symbol link to /run

also, I'd suggest follow the convention to rename the file to throttled.initd to make it explicitly as openrc init script..

Reply:
file throttled.initd?